packages/gatsby/src/utils/websocket-manager.js

```diff
@@ -103,8 +103,8 @@ class WebsocketManager {
     this.pageResults = new Map()
     this.staticQueryResults = new Map()
     this.errors = new Map()
-    this.websocket // eslint-disable-line no-unused-expressions
-    this.programDir // eslint-disable-line no-unused-expressions
+    // this.websocket
```

Comment:
Not sure if this is the best solution. The variable would be undefined on this context.